src/pages/kwonojae/Main/components/Feed.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons';
+import {
+  faPaperPlane,
+  faHeart,
+  faComment,
+  faBookmark,
+} from '@fortawesome/free-regular-svg-icons';
+import Writecomment from './writecomment';
+import './Feed.scss';
+
+export default class Feed extends Component {
+  constructor() {
+    super();
+    this.state = {
+      feedList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/kwonojae/Feeddata.json', {
```

Comment:
```suggestion
    fetch('/data/kwonojae/Feeddata.json', {
```
이런식으로 처리해주시면 현재 웹서버가 켜져있는 주소 기준으로 접근하기때문에 나중에 서버 주소가바뀌거나
포트번호가 바뀌거나했을때 자연스럽게 대응이 됩니다.

Reply:
네 어제 세션듣고 해당 부분들 수정했습니다